src/lein_monolith/dependency.clj

```diff
@@ -165,6 +165,74 @@
           (recur (conj result node)
                  (into (pop queue) (set/difference consumers result))))))))
 
+(defn unique-cycles
+  "Return a set of all unique cycles in dependency graph m."
+  [m]
+  {:pre [(map? m)]
+   :post [(set? %)]}
+  (let [path->cycles (fn path->cycles [path]
+                       {:pre [(seq path)]}
+                       (let [k (peek path)
+                             mks (set (keys m))
+                             vs (m k)
+                             path-set (set path)]
+                         (mapcat (fn [v]
+                                   (cond
+                                     ; found the cycle
+                                     (path-set v) [(into []
```

Comment:
This `cond` only has one clause and the `:else` default, so it should be a regular `if`.